cmd/deploy/status.go

```diff
@@ -85,7 +85,9 @@ func NewDeployStatusCmd(configuration *config.Configuration) *cobra.Command {
 			return status(cmd, configuration, deploymentId)
 		},
 	}
-	cmd.Flags().StringVarP(&deploymentId, "deploymentId", "i", "", "(Required) The ID of an existing deployment.")
+	cmd.Flags().StringVarP(&deploymentId, "deploymentId", "i", "", "(Required) The ID of an existing deployment.\n"+
+		"deploymentId can be found by navigating to the deployment status page and looking in the url: \n"+
+		"https://console.cloud.armory.io/deployments/pipeline/<deploymentId>")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&deploymentId, "deploymentId", "i", "", "(Required) The ID of an existing deployment.\n"+
		"You can find the deploymentId by navigating to the deployment status page and looking in the URL: \n"+
		"https://console.cloud.armory.io/deployments/pipeline/<deploymentId>")
```

Change passive to active